learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -15,70 +15,91 @@ redirect_from:
 
 ## Setting up the prerequisites
 
-To complete this tutorial, you need:
+>**Info:** You need a command terminal to complete this tutorial.
 
-1. A command terminal
-2. A text editor
-    >**Tip:** Preferably, [Visual Studio Code](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed.
-3. A [Ballerina installation](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/)
+To set up the other prerequisites, follow these steps:
+
+1. Install [Ballerina](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/).
+2. Install a text editor. 
+  >**Tip:** Preferably, [Visual Studio Code](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed.
+
+## Understanding the implementation
```

Comment:
We need to avoid  using `ing` for titles. 
```
For a task-based heading, start with a [bare infinitive](https://wikipedia.org/wiki/Infinitive#English), also known as a plain form or [base form](https://wikipedia.org/wiki/English_verbs#Base_form) verb.
```
Refer [1] https://developers.google.com/style/headings?hl=en

Reply:
Done.